blueprints/app/files/package.json

```diff
@@ -37,6 +37,7 @@
     "ember-export-application-global": "^1.0.4",
     "ember-load-initializers": "^0.5.0",
     "ember-resolver": "^2.0.3",
+    "ember-welcome-page": "locks/ember-welcome-page#master",
```

Comment:
we will need this to be from npm, before we can merge. Firewall users, often have issues with direct github deps